src/java.base/share/classes/java/time/LocalDateTime.java

```diff
@@ -71,6 +71,7 @@
 import static java.time.LocalTime.NANOS_PER_SECOND;
 import static java.time.LocalTime.SECONDS_PER_DAY;
 import static java.time.temporal.ChronoField.NANO_OF_SECOND;
+import static jdk.internal.util.DateTimeHelper.formatTo;
```

Comment:
Do not import static across package boundaries. I discourage it even within the same package.
It makes the code hard to read and follow where the function is coming from.